homework/index.js

```diff
@@ -30,13 +30,127 @@
+        alt: 'contributor avatar',
+        class: 'contributor-avatar',
+      });
+      const contributorInfoContainer = createAndAppend('div', contributorContainer, {
+        class: 'contributor-info-container',
+      });
+      const contributorNameContainer = createAndAppend('div', contributorInfoContainer, {});
+      const contributorName = createAndAppend('h3', contributorNameContainer, {});
+      createAndAppend('a', contributorName, {
+        text: element.login,
+        href: element.html_url,
+        target: '_blank',
+      });
+      const contributorNumberOfContributionsContainer = createAndAppend(
+        'div',
+        contributorInfoContainer,
+        {},
+      );
+      createAndAppend('h3', contributorNumberOfContributionsContainer, {
+        text: element.contributions,
+      });
+    });
+  }
+
+  function fetchRepositoryContributors(url, containerDiv) {
+    fetchJSON(url, (err, data) => {
+      if (err) {
+        createAndAppend('h2', containerDiv, { text: err.message, class: 'alert-error' });
+      } else {
+        buildRepositoryContributors(data, containerDiv);
+      }
+    });
+  }
+
+  function buildRepositoryInfoAndFetchContributors(
+    repositoryObject,
+    infoContainer,
+    contributorsContainer,
+  ) {
+    buildRepositoryInfo(repositoryObject, infoContainer);
+    fetchRepositoryContributors(repositoryObject.contributors_url, contributorsContainer);
+  }
+
+  function startUpAndBuildSelectList(arrayOfRepositories, divContainer) {
+    const leftContainer = createAndAppend('div', divContainer, { id: 'left-container' });
+    const rightContainer = createAndAppend('div', divContainer, { id: 'right-container' });
+    createAndAppend('img', leftContainer, { src: './hyf.png', id: 'hyf-logo', alt: 'logo image' });
+    createAndAppend('p', leftContainer, { text: '"Refugee code school in Amsterdam"' });
+    createAndAppend('h4', leftContainer, { text: 'Select a repository to display information:' });
+    const selectMenu = createAndAppend('select', leftContainer, { id: 'select-menu' });
+    arrayOfRepositories.sort((a, b) => a.name.localeCompare(b.name, 'en', { sensitivity: 'base' }));
+
+    arrayOfRepositories.forEach((repository, index) =>
+      createAndAppend('option', selectMenu, { text: repository.name, value: index }),
+    );
+
+    const repositoryInfoSection = createAndAppend('div', leftContainer, {
+      id: 'repository-info-section',
+    });
+
+    buildRepositoryInfoAndFetchContributors(
+      arrayOfRepositories[0],
+      repositoryInfoSection,
+      rightContainer,
+    );
+
+    selectMenu.addEventListener('change', event => {
+      repositoryInfoSection.innerHTML = '';
+      rightContainer.innerHTML = '';
+
+      buildRepositoryInfoAndFetchContributors(
+        arrayOfRepositories[event.target.value],
+        repositoryInfoSection,
+        rightContainer,
+      );
+    });
+  }
+
   function main(url) {
     fetchJSON(url, (err, data) => {
```

Comment:
The data we get is a list of repositories, so you could rename `data` to `repositories`.